tests/testthat/test-TextType.R

```diff
@@ -49,3 +49,25 @@ testthat::test_that(
     unlink(out)
   }
 )
+
+
+
+testthat::test_that(
+  "We can parse a richer abstract into markdown",
+  {
+    skip_on_cran()
+    skip_on_appveyor()
+    skip_on_os("windows")
```

Comment:
oops not sure what happened when I updated my fork, currently I'm running the tests on my PC and then I'll add the modification to this PR.